.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "printWidth": 90,
```

Comment:
For me maybe 90 chars is a bit tight, but if it is our standard I'm ok with this.